src/cli/self_update.rs

```diff
@@ -242,19 +212,9 @@ fn pixi_binary_name() -> String {
     format!("pixi{}", std::env::consts::EXE_SUFFIX)
 }
 
-fn default_pixi_binary_path() -> std::path::PathBuf {
-    dirs::home_dir()
-        .expect("Could not find the home directory")
-        .join(".pixi")
-        .join("bin")
-        .join(pixi_binary_name())
-}
-
-// check current binary is in the default pixi location
-fn is_pixi_binary_default_location() -> bool {
-    same_file::is_same_file(
-        std::env::current_exe().expect("Failed to retrieve the current pixi binary path"),
-        default_pixi_binary_path(),
+pub async fn execute_stub(_: Args) -> miette::Result<()> {
+    let message = option_env!("PIXI_SELF_UPDATE_DISABLED_MESSAGE");
```

Comment:
Whoops, I think you are not yet using the `message`, right? :)

Reply:
I'm using in in the line below this with `message.unwrap_or`, which returns `message` if it is not `None` and the passed string otherwise :wink:

Reply:
Ah sorry, I did totally overlook that. Excellent!